conda-forge.yml

```diff
@@ -9,3 +9,4 @@ github:
 provider:
   win: azure
 test_on_native_only: true
+provider: {linux_aarch64: default}
```

Comment:
are you sure this is the right syntax?

Reply:
https://conda-forge.org/docs/maintainer/conda_forge_yml.html#linux-aarch64

Reply:
I followed the instructions from [here](https://thepracticalsysadmin.com/building-multiarch-conda-forge-recipes/). In e.g. the [Faunus feedstock](https://github.com/conda-forge/faunus-feedstock/blob/main/conda-forge.yml) `default` however `azure`. Nonetheless, aarch64 builds are triggered under Checks. No expert here, just trying to solve the problem that I cannot use `mdtraj` in aarch64-linux Docker images on apple silicon.